### ISSUE TITLE
f.lux issue

### ISSUE THREAD

**Issue by carlodrift** [MonitorControl/MonitorControl#1210]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I keep getting the popup that tells me to close the other applications that manage the brightness except that I don't use any of them 

### Expected behavior

No popup I guess?

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.6 / 13
- Mac model: Macbook Air M1
- MonitorControl version: 4.1.0 build 7034
- Monitor(s): iiyama prolite ge2488hs
- Monitor Cable(s)/Connection(s): hdmi
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
You can ask the app not to warn about it again - but if you have this popup, then good chance you have some other conflicting app running (BetterDisplay, f.lux, etc). If not, then I have no idea why this is happening (the prompt is shown when the expected color table set by the app differs from what the app actually finds).

You can try using [BetterDisplay](http://betterdisplay.pro/) if this is a returning issue as in that app I did not implement a warning like this. :)

(note: if you use BetterDispaly or MonitorControl via HDMI, do not forget to disable DDC brightness control as that does not work through the HDMI port of M1 macs).

**Comment by carlodrift** [comment]
Should I disable DDC brightness if my display is connected to HDMI via a USB-C hub?

**Comment by waydabber** [comment]
Ohh sorry, I see you have an M1 Air. No, the problem is only for built-in HDMI ports, but an Air does not have one. If your dock properly translates DDC i2c communication over HDMI (most do) then you should not disable it. :) My bad!

**Comment by carlodrift** [comment]
I don't have any other applications that affect brightness or screens. Maybe it's the screen's fault? When it turns on I sometimes have to move the slider a bit to get the right brightness

**Comment by waydabber** [comment]
I don't think so, this specific warning is only happens when there is a change caused in the color table (this defines how each colors look on your screen) by a third party app.

**Comment by waydabber** [comment]
Unfortunately not. What you might want to do is to make sure no apps are launched (either as a Login Item or as an agent) and then start them one-by-one and see when the issue appears. But not many apps should do color table manipulation so if you figure out a new app that is interfering, let me know so I can add it to the bad boys list lol. :)

**Comment by carlodrift** [comment]
I use Night Shift and True Tone from macOS, is this a problem?

**Comment by waydabber** [comment]
No, those should be applied over the color table manipulated screen and should not interfere.
